source/develop/release-management/features/virt/headless-vm.html.md

```diff
@@ -86,11 +86,65 @@ In case of Headless VM / Template / Pool / Instance Type:
 
 No changes in the DB
 
+#### REST API
+
+There is currently a  partial support for Headless in OVirt APIs.
```

Comment:
- there are two spaces after the "currently a"
- for headless VMs in oVirt
- the oVirt is written with small o :)